docs/qa/CometBFT-QA-38.md

```diff
@@ -0,0 +1,187 @@
+choosen `c=2,r=200`, which has a transaction load equal to `c=1,r=400`.
+
+## Latencies
+
+The following figure plots the latencies of the experiment carried out with the
+configuration `c=1,r=400`.
+
+![latency-1-400](img38/200nodes/e_de676ecf-038e-443f-a26a-27915f29e312.png).
+
+For reference, the following figure shows the latencies of one of the
+experiments in the baseline, where `c=2,r=200` is equivalent to the same
+configuration used in this experiment. As can be seen, latencies are quite
+similar, and even in some cases the baseline has slightly higher latencies.
+
+![latency-2-200-37](img37/200nodes_cmt037/e_75cb89a8-f876-4698-82f3-8aaab0b361af.png)
+
+## Prometheus Metrics on the Chosen Experiment
+
+This section further examines key metrics for this experiment extracted from
+Prometheus data regarding the chosen experiment with configuration `c=1,r=400`.
+
+### Mempool Size
+
+The mempool size, a count of the number of transactions in the mempool, was
+shown to be stable and homogeneous at all full nodes. It did not exhibit any
+unconstrained growth. The plot below shows the evolution over time of the
+cumulative number of transactions inside all full nodes' mempools at a given
+time.
+
+![mempoool-cumulative](img38/200nodes/mempool_size.png)
+
+The following picture shows the evolution of the average mempool size over all
+full nodes, which mostly oscilates between 1000 and 2500 outstanding
+transactions.
+
+![mempool-avg](img38/200nodes/avg_mempool_size.png)
+
+The peaks observed coincide with the moments when some nodes reached round 1 of
+consensus (see below).
+
+The behavior is similar to the observed in the baseline, presented next.
+
+![mempool-cumulative-baseline](img37/200nodes_cmt037/mempool_size.png)
+
+![mempool-avg-baseline](img37/200nodes_cmt037/avg_mempool_size.png)
+
+
+### Peers
+
+The number of peers was stable at all nodes. It was higher for the seed nodes
+(around 140) than for the rest (between 20 and 70 for most nodes). The red
+dashed line denotes the average value.
+
+![peers](img38/200nodes/peers.png)
+
+Just as in the baseline, shown next, the fact that non-seed nodes reach more
+than 50 peers is due to [\#9548].
+
+![peers](img37/200nodes_cmt037/peers.png)
+
+
+### Consensus Rounds per Height
+
+Most heights took just one round, that is, round 0, but some nodes needed to
+advance to round 1.
+
+![rounds](img38/200nodes/rounds.png)
+
+The following specific run of the baseline required some nodes to reach round 1.
+
+![rounds](img37/200nodes_cmt037/rounds.png)
+
+
+### Blocks Produced per Minute, Transactions Processed per Minute
+
+The following plot shows the rate in which blocks were created, from the point
+of view of each node. That is, it shows when each node learned that a new block
+had been agreed upon.
+
+![heights](img38/200nodes/block_rate.png)
+
+For most of the time when load was being applied to the system, most of the
+nodes stayed around 20 blocks/minute.
```

Comment:
You may want to mention that the last part of the graph does contain load, hence the block production rate increases slightly. Check the X axis: this graph is using the "extended window"

Reply:
Sorry, I typed this too soon... you explain it perfectly in the next paragraph